### ISSUE TITLE
mysql max connections errors

### ISSUE THREAD

**Issue by bwellsnc** [yaoweibin/nginx_tcp_proxy_module#29]
I have the proxy working now, but I have a new issue.  I created a simple test proxy to an internal mysql server I have.  It worked fine for the first couple of connections, then I received this error:

'hostname' is blocked because of many connection errors. Unblock with 'mysqladmin flush-hosts'

By default mysql is set to 10 max connection errors.  In order to compensate, I increased the max_connection_errors variable in mysql to 5000.  I really didn't want to make many changes on the MySQL side.  Below is what I have in nginx:

check interval=3000 rise=2 fall=5 timeout=1000 type=mysql;

This is the default with the type of mysql added.  I would like to know how I can fix this in nginx vs making changes in mysql configuration.  If it can't be fixed in nginx, then I will just stick to the changes in mysql for now.  Thanks!
